ArkEcosystem/explorer
# fix: monitor not updating when blocks are missed

## Summary

https://app.clickup.com/t/mt6pr7

To test:

- make sure delegate monitor works correctly
- when a delegate misses blocks, it should update the status as and when the block is missed, not when the next delegate forges (see task)

**This also moves debug code to a unit test which is used to validate core logic for the delegate tracker**







## Checklist
